test_rclcpp/test/node_check_names.cpp

```diff
@@ -12,7 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <algorithm>
 #include <string>
+#include <utility>
```

Comment:
I'm not sure if this header is being used explicitly in this file.

Reply:
I'm not sure either. I'll cut it and see if it breaks.